content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -29,22 +29,20 @@ Prisma Client provides the following metrics:
   - `prisma_client_queries_total`: The total number of Prisma Client queries executed.
   - `prisma_datasource_queries_total`: The total number of datasource queries executed (SQL queries in relational databases, and commands in MongoDB).
     - The value returned by `prisma_datasource_queries_total` can be greater than `prisma_client_queries_total`, because some Prisma Client operations create multiple queries.
-  - `prisma_pool_connections_opened_total`: The total number of [pool connections](/concepts/components/prisma-client/working-with-prismaclient/connection-pool#default-connection-pool-size) opened.
+  - `prisma_pool_connections_open`: The total number of [pool connections](/concepts/components/prisma-client/working-with-prismaclient/connection-pool#default-connection-pool-size) opened.
   - `prisma_pool_connections_closed_total`: The total number of pool connections closed.
 
 - Gauges (can increase or decrease):
 
-  - `prisma_pool_connections_open`: The number of currently open pool connections.
+  - `prisma_client_queries_active`: The number of currently active Prisma Client queries.
+  - `prisma_client_queries_wait`: The number of Prisma Client queries currently waiting for a connection because all connections are in use.
   - `prisma_pool_connections_busy`: The number of currently busy pool connections. These pool connections are currently executing a datasource query.
   - `prisma_pool_connections_idle`: The number of pool connections that are not currently being used. These pool connections are waiting for the next datasource query to run.
-  - `prisma_client_queries_active`: The number of currently active transactions.
-  - `prisma_client_queries_wait`: The number of Prisma Client queries currently waiting for a connection because all connections are in use.
+  - `prisma_pool_connections_opened_total`: The number of currently open pool connections.
```

Comment:
This is still a counter (indicated by the `_total`).

Reply:
This was included in the gauge, based on the output I got. Perhaps it's a bug? I'll retry it later and get back to you on this.